evm/utils/kdf.ts

```diff
@@ -0,0 +1,95 @@
+import bs58 from "bs58"
```

Comment:
Did you write this code from scratch, or is there an original source you referenced? If there is an original source, could you add a link to it in the comments? I think it could be helpful.

Reply:
This has been implemented by @kiseln in previous commits, so maybe he can share the links

Reply:
https://github.com/near-examples/near-multichain/blob/main/src/services/kdf.js

Reply:
1df4370a93d78233196a8178bc764d5b13687aae